src/components/Navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import LoginBtn from './LoginBtn';
+import { useTranslation } from 'next-i18next';
+import type { GetStaticProps, InferGetStaticPropsType } from 'next';
+import { getStaticProps } from '@/pages';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+
+ //lines below will be needed fo authentication
+// interface _Props {
+//   authData: {
+//     user?: {
+//       name?: string | null | undefined;
+//       email?: string | null | undefined;
+//       image?: string | null | undefined;
+//     };
+//   } | null;
+//   authStatus: 'authenticated' | 'loading' | 'unauthenticated';
+// }
```

Comment:
❗ 
remove these commented lines, when you implement the auth you can add them

this PR has nothing to do with auth, so it is better if it has nothing to do with auth

Reply:
OK emoved